demo/simple_test.yarn

```diff
@@ -0,0 +1,48 @@
+title: Start
+tags: 
+colorID: 0
+position: 738,494
+---
+<<set $cake to "cake">>
+
+Linda: Why hello there.
+Linda: I'd love to have some "[color=blue]{$cake}[/color]" with you.
+
+<<showcake true>>
+
+Bob: That's terrifying...
+Bob: Please put that away.
+Bob: That's not even {$cake}... it's
+
+-> Pizza
+  <<set $cake to "pizza">>
+  Bob: It's {$cake}...
+-> Taco Salad
+  <<set $cake to "taco salad">>
+  Bob: It's {$cake}...
+-> Actually it is...
+  Bob: Oh. No wait. That is {$cake}...
+  [[But. Seriously Lin... put the {$cake} away|noCake]]
+  [[I guess it's OK then...|yayCake]]
+
+===
+title: noCake
+tags: 
+colorID: 0
+position: 738,494
+---
+
+Linda: Aww. You're no fun.
+<<showcake false>>
+
+Bob: oh thank god.
+===
+title: yayCake
+tags: 
+colorID: 0
+position: 738,494
+---
+
+Linda: Yay! I love ${cake}!
```

Comment:
The symbol was switched around, should be `{$cake}` :D

Reply:
Well... I guess that's years of JavaScript and a touch of dyslexia will do to you. 🤣